apps/condo/domains/common/utils/serverSchema/index.js

```diff
@@ -0,0 +1,25 @@
+
+const loadModelsByChunks = async ({
+    context,
+    model,
+    where = {},
+    sortBy = ['createdAt_ASC'],
+    chunkSize = 100,
+    limit = 100000,
+}) => {
+    let skip = 0
+    let maxiterationsCount = Math.floor(limit / chunkSize)
```

Comment:
Are you sure about `Math.floor(limit / chunkSize)`?
For example, a query of 199 items, by 100 items in one chunks requires 2 queries.
We have 199/100 = 1.99 and floored — `1`.

Condition in `while` using `--` prefixed decrement, will get false and we will never have second query.

I think, `Math.ceil` should be used in this case.